scripts/tasks.js

```diff
@@ -19,10 +19,11 @@ exports.replaceWebpack = () => {
 
 exports.copyAssets = (type, outputDir) => {
   const env = type === 'build' ? 'prod' : type
-  rm('-rf', outputDir)
-  mkdir(outputDir)
-  cp(`app/manifest.${env}.json`, `${outputDir}/manifest.json`)
-  cp('-R', 'app/_locales/', outputDir)
-  cp('-R', 'app/assets/*', outputDir)
-  exec(`pug -O "{ env: '${env}' }" -o ${outputDir} app/views/`)
+  const dir = type === 'build' ? outputDir : type
```

Comment:
it's very confusing to me that we pass in `type` and `outputDir` and then potentially modify both of them based on the value of the other one. Why don't we just change `type` to `env` and pass in the correct values to start with?